packages/xrpl/snippets/src/paths.ts

```diff
@@ -15,7 +15,7 @@ async function createTxWithPaths(): Promise<void> {
     issuer: 'rVnYNK9yuxBz4uP8zC8LEFokM2nqH3poc',
   }
 
-  const request = {
+  const resp = await client.request({
     command: 'ripple_path_find',
```

Comment:
It's actually bad practice to use this command, we should be recommending that people use the `path_find` subscription instead

Reply:
https://github.com/XRPLF/xrpl.js/issues/2385